/**: 
  controller_manager:
    ros__parameters:
      update_rate: 100  # Hz

      arm_controller:
        type: joint_trajectory_controller/JointTrajectoryController

#      arm_joint_state_broadcaster:
#        type: joint_state_broadcaster/JointStateBroadcaster

  arm_controller:
    ros__parameters:
      use_sim_time: True
      joints:
        - joint_0
      command_interfaces:
        - position
      state_interfaces:
        - position
        - velocity